docs/scos/dev/upgradability-services/new-documentation/unique-entity-name-not-unique.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Entity name is not unique
+description: Reference information for evaluator and upgrader tools.
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+## Entity name is not unique
+
+The names of the following entities should be unique:
+
+* Transfers
+* Transfer properties
+* Database tables
+* Database columns
+* Methods
+* Constants
+
+If a minor or major release introduces an entity with the same name, the entity in your project might change behavior or cause issues.
```

Comment:
The same, as to me we need to split it per groups, like DTO, DB, etc.

Reply:
Did not get it. 
Lets discuss